app/controllers/lists_controller.rb

```diff
@@ -2,7 +2,11 @@ class ListsController < ApplicationController
 #before_action :find_list, only: [:show, :update, :edit, :destroy]
 
   def index
-    @lists = List.all.order("created_at DESC")
+    @lists = List.all
```

Comment:
Out of curiosity, why did you remove this order?

Reply:
I was afraid that it was a reason for my inability to create List items. I will put it back in since that didn't seem to matter.